src/Compilers/Core/Portable/CommandLine/AnalyzerConfig.cs

```diff
@@ -0,0 +1,485 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.IO;
+using System.Text.RegularExpressions;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Roslyn.Utilities;
+using AnalyzerOptions = System.Collections.Immutable.ImmutableDictionary<string, string>;
+using TreeOptions = System.Collections.Immutable.ImmutableDictionary<string, Microsoft.CodeAnalysis.ReportDiagnostic>;
+
+namespace Microsoft.CodeAnalysis
+{
+    /// <summary>
+    /// Represents a single EditorConfig file, see http://editorconfig.org for details about the format.
+    /// </summary>
+    public sealed partial class AnalyzerConfig
```

Comment:
It is. Git doesn't want to show that because the file was also changed, so the first commit just does the rename. You can review commit-by-commit if you want and all the changes should be delineated.